lnst/Recipes/ENRT/VirtualBridgeVlansOverBondRecipe.py

```diff
@@ -40,6 +40,8 @@ class VirtualBridgeVlansOverBondRecipe(VlanPingEvaluatorMixin,
     guest4 = HostReq()
     guest4.eth0 = DeviceReq(label="to_guest4")
 
+    vlan_ids = Param(default=[10, 20])
```

Comment:
i think it would probably be better if this was 2 parameters instead of a single parameter that expects a list of 2 numbers.

Right now you could do:

`VirtualBridgeVlansOverBondRecipe(vlan_ids=[123])` which would cause a traceback during test configuration

or

`VirtualBridgeVlansOverBondRecipe(vlan_ids=[1, 2, 3])` which just wouldn't use the value `3` and that could potentially be confusing?

instead if we had:
```
vlan1_id = IntParam(default=10)
vlan2_id = IntParam(default=20)
```
It resolves both situations and is potentially more clear? What do you think?

Another option would be to dynamically create as many vlans as the number of values in the `vlan_ids` list but at this point that's an unnecessarily big change to the recipes...

Reply:
I tried to use the same names everywhere (vlan_id for single integer and vlan_ids for arrays) for consistency and ease to use. I though in the case you provide a wrong number of elements, but I though that this won't be used very often, and if you use you probably know what you're doing.
But anyway, your suggestion seems good to me: typechecking, avoid backtraces, number of required IDs more clearly indicated, naming still consistent...

Reply:
Done